### DISCUSSION TITLE
Welcome to openalgo Discussions!

### DISCUSSION THREAD

**Discussion by marketcalls** [marketcalls/openalgo#73] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anup-kr** [comment]
Major chunk of work has gone in last 6 months, meaning it's relatively new repo. Why picked flask framework, given FastApi is around with better async support? I am not expert in either but I am curious to know the backings behind choosing this framework. The system overall is decently I/O heavy(call to fetch marketdate, call to broker etc). Could FastApi have helped here, as it inherently supports asyncio? Thanks in advance.

**Comment by marketcalls** [comment]
You're absolutely right that FastAPI offers excellent async support and would be a solid choice for I/O-heavy workloads like fetching market data and interacting with brokers. In fact, we're big fans of FastAPI ourselves and have used it in other parts of our ecosystem.

Flask has a vast ecosystem, mature plugin support, and a large user base. It allowed us to move faster initially, especially given the project's target audience includes traders who may want to extend the platform without deep backend experience.

**Comment by SujJiGoEnka** [comment]
can we add stoploss and target by using chartink?

**Comment by Harsshit28** [comment]
can you add backtester
